src/app/App.js

```diff
@@ -8,10 +8,24 @@ import {
   saveToLocalStorage,
   getFromLocalStorage
 } from '../services'
+import ActivityList from '../components/ActivityList'
 
 const GlobalStyle = createGlobalStyle`
-  /* Global Styles go here */
-`
+:root {
+  --primary-color: #2E8B57;
+  --grey: rgba(216, 216, 214, 0.44);
+  --yellow: #FDE100;
+  --blue: #0072C2;
+  --bg-grey: #D8D8D8;
+  --light-font: #FFFFFF;
+  --dark-font: #000000;
+  --red-font:#DF4D60;
+}
+
+body {
+    font-family: -apple-system, BlinkMacSystemFont, sans-serif;
+    font-size: 22px;
+}`
 
 const Grid = styled.div`
   xdisplay: grid;
```

Comment:
seems like you don't need this one here you can delete it (for now)

Reply:
will be deleted.